validation/src/test/java/no/bibsys/entitydata/validation/DataValidatorTest.java

```diff
@@ -16,17 +22,13 @@
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 
-import java.io.IOException;
-import java.nio.charset.StandardCharsets;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-
-import static org.hamcrest.Matchers.containsString;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import no.bibsys.entitydata.validation.exceptions.EntityFailedShaclValidationException;
+import no.bibsys.utils.IoUtils;
+import no.bibsys.utils.ModelParser;
 
 public class DataValidatorTest extends ModelParser {
 
+    private static final String DELIMITER = "";
```

Comment:
This is a bit weird, and should at least be EMPTY_STRING = ""; DELIMITER = EMPTY_STRING;